input.go

```diff
@@ -344,6 +363,66 @@ func (w *Window) SetCursorPosition(xpos, ypos float64) error {
 	return fetchError()
 }
 
+// Creates a new custom cursor image that can be set for a window with SetCursor.
+// The cursor can be destroyed with Destroy. Any remaining cursors are destroyed by Terminate.
+//
+// The pixels are 32-bit little-endian RGBA, i.e. eight bits per channel. They are arranged
+// canonically as packed sequential rows, starting from the top-left corner.
+// 
+// All non-RGBA images will be converted to RGBA.
+//
+// The cursor hotspot is specified in pixels, relative to the upper-left corner of the cursor image.
+// Like all other coordinate systems in GLFW, the X-axis points to the right and the Y-axis points down.
+func CreateCursor(img image.Image, xhot, yhot int) (*Cursor, error) {
```

Comment:
I'm still thinking whether it makes a better API/experience to accept `image.Image` instead of just `imageData []byte`...